contracts/test/integration/IntegrationTestMakerRouter.t.sol

```diff
@@ -0,0 +1,351 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+import {DSTest} from "../unit/utils/DSTest.sol";
+import {Vm} from "../unit/utils/Vm.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {IDSSPSM} from "../../pcv/maker/IDSSPSM.sol";
+import {MakerRouter} from "../../pcv/maker/MakerRouter.sol";
+import {PegStabilityModule} from "../../peg/PegStabilityModule.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IVolt} from "../../volt/IVolt.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {Core} from "../../core/Core.sol";
+import {Constants} from "../../Constants.sol";
+
+contract IntegrationTestMakerRouter is DSTest {
+    using SafeCast for *;
+
+    MakerRouter private makerRouter;
+
+    Core private core = Core(MainnetAddresses.CORE);
+    IDSSPSM private daiPSM = IDSSPSM(MainnetAddresses.MAKER_DAI_USDC_PSM);
+    PegStabilityModule private feiPSM =
+        PegStabilityModule(MainnetAddresses.FEI_DAI_PSM);
+
+    IERC20 private dai = IERC20(MainnetAddresses.DAI);
+    IVolt private fei = IVolt(MainnetAddresses.FEI);
+    IERC20 private usdc = IERC20(MainnetAddresses.USDC);
+
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    uint256 public constant USDC_SCALING_FACTOR = 1e12;
+
+    uint256 public constant mintAmount = 200_000_000e18;
+
+    function setUp() public {
+        makerRouter = new MakerRouter(MainnetAddresses.CORE, daiPSM, feiPSM);
```

Comment:
This integration test should leverage the already deployed maker router instead of deploying another one.

Reply:
Similar comment as here in regards to this being a test prior to deployment https://github.com/volt-protocol/volt-protocol-core/pull/97#discussion_r952225881

Reply:
contracts have already been deployed, please check MainnetAddresses.sol